charts/application/values.yaml

```diff
@@ -351,6 +351,11 @@ sidecars: []
 #   lifecycle:
 #     postStart: {}
 #     preStop: {}
+#   # ports of the sidecar container to be published by the pod (e.g. for prometheus exporter sidecars
```

Comment:
```suggestion
#   # ports of the sidecar container to be published by the pod, e.g. for prometheus exporter sidecars
```
or
```suggestion
#   # ports of the sidecar container to be published by the pod (e.g. for prometheus exporter sidecars)
```

Reply:
thx. will change, if that's not the only thing, as Marius is waiting for it in a blocking manner :P